Rakefile

```diff
@@ -12,4 +12,12 @@ RSpec::Core::RakeTask.new(:junit) do |t|
   t.pattern    = 'spec/'
 end
 
+desc 'run static analysis with rubocop'
+task(:rubocop) do
+  require 'rubocop'
+  cli = RuboCop::CLI.new
+  exit_code = cli.run(%w(--display-cop-names --format simple))
+  raise "RuboCop detected offenses" if exit_code != 0
+end
```

Comment:
What if I told you that Rubocop has Rake shims to help you out here?

From https://github.com/mckern/vanagon/blob/master/Rakefile:

```ruby
desc 'Run RuboCop'
RuboCop::RakeTask.new(:rubocop) do |task|
  task.options << '--display-cop-names'
end
```

That'll give you:

```bash
mckern@flexo vanagon (git:master) $ bundle exec rake -T | grep rubo
rake rubocop               # Run RuboCop
rake rubocop:auto_correct  # Auto-correct RuboCop offenses
mckern@flexo vanagon (git:master) $
```